datadog_checks_dev/datadog_checks/dev/tooling/templates/docs/example_nagios.yaml

```diff
@@ -41,8 +44,8 @@ files:
             | Parameter            | Value                                        |
             | -------------------- | -------------------------------------------- |
             | `<INTEGRATION_NAME>` | `nagios`                                     |
-            | `<INIT_CONFIG>`      | blank or `{}`                                |
-            | `<INSTANCE_CONFIG>`  | `{"nagios_conf": "/etc/nagios3/nagios.cfg"}` |
+            | `<INIT_CONFIG>`      | blank or `{{}}`                                |
+            | `<INSTANCE_CONFIG>`  | `{{"nagios_conf": "/etc/nagios3/nagios.cfg"}}` |
```

Comment:
Since these description blocks are processed with the Python `format` method, any literal braces need to be escaped to translate appropriately.